src/material/slider/slider.scss

```diff
@@ -115,23 +115,23 @@ $mat-slider-horizontal-margin: 8px !default;
         @include token-utils.create-token-slot(width, value-indicator-width);
         @include token-utils.create-token-slot(transform, value-indicator-text-transform);
       }
-    }
 
-    // The `.mat-ripple` wrapper here is redundant, but we need it to
-    // increase the specificity due to how some styles are setup in g3.
-    .mat-ripple {
-      // These ripple colors are custom so we have to introduce our own tokens.
-      .mat-ripple-element {
-        background-color: var(--mat-mdc-slider-ripple-color, transparent);
-      }
+      // The `.mat-ripple` wrapper here is redundant, but we need it to
+      // increase the specificity due to how some styles are setup in g3.
+      .mat-ripple {
+        // These ripple colors are custom so we have to introduce our own tokens.
+        .mat-ripple-element {
+          @include token-utils.create-token-slot(background-color, ripple-color);
+        }
 
-      .mat-mdc-slider-hover-ripple {
-        background-color: var(--mat-mdc-slider-hover-ripple-color, transparent);
-      }
+        .mat-mdc-slider-hover-ripple {
+          @include token-utils.create-token-slot(background-color, hover-ripple-color);
```

Comment:
In other components (haven't checked slider), MDC uses terminology like `hover-state-layer` for the hover/focus indication. Should we use something similar?

Reply:
Would `ripple-color`, `ripple-hover-state-layer`, and `ripple-focus-state-layer` be correct?

Reply:
I think the terminology is "state-layer" instead of "ripple". E.g. in the button we have `hover-state-layer-opacity` and `focus-state-layer-opacity`.

Reply:
Looks like now it says both "ripple" and "state layer".

Reply:
Does "state layer" = "ripple" in this terminology? I'm not sure what "state layer" actually refers to so I'm unsure what the correct name for these tokens would be

Reply:
I think what they call a "state layer" is an overlay on top of the component. E.g. the button has a semi-transparent layer when it's focused.

Reply:
So for our case the "state layer" is the ripple, correct? So the correct names for these tokens would be `state-layer-color`, `hover-state-layer-color`, and `focus-state-layer-color`?